Doc/reference/compound_stmts.rst

```diff
@@ -1368,6 +1368,8 @@ access the local variables of the function containing the def.  See section
       Support for forward references within annotations by preserving
       annotations in a string form at runtime instead of eager evaluation.
 
+   :pep:`318` - Decorators for Functions and Methods
+      Function and method decorators were introduced.
```

Comment:
LGTM as well, thanks. Optionally, it might be useful to mirror the "Class definitions" section and also reference PEP 3129 under the "Function definitions" block with something like

```suggestion
      Function and method decorators were introduced. Class decorators were introduced in :pep:`3129`.
```

I'll let Hugo decide whether that addition's useful or not 🙂 

Since we're changing this section anyways, on line 1365 there's a missing period after `variables and instance variables` that my suggestion can't quite reach. If you could give that a quick fix, that'd be great too!

Reply:
Sure, let's include this.

@Rasputin2 Please could you add that? You can click the "Commit suggestion" button to apply this.

And add the period as a new commit in this PR, rather than a new one.

Thanks!